Xamarin.PropertyEditing.Mac/Controls/Custom/EmptyBrushEditorViewController.cs

```diff
@@ -0,0 +1,28 @@
+using AppKit;
+using CoreGraphics;
+using Xamarin.PropertyEditing.Drawing;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	class EmptyBrushEditorViewController : NotifyingViewController<BrushPropertyViewModel>
+	{
+		public EmptyBrushEditorViewController ()
+		{
+			PreferredContentSize = new CGSize (100, 100);
+		}
+
+		NSButton brushEditor;
+
+		public override void LoadView ()
+		{
+			View = brushEditor = new NSButton {
+				Title = "Edit"
```

Comment:
Needs to be localized.

Reply:
Didn't really have a design here, could also do like windows an leave it empty.